### PR TITLE
Add Gridsome example

### PR DESCRIPTION
**Pull request by timothyis**
This pull request adds an example Gridsome application, ready to be deployed with Now 2.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@
 | [HTML Minifier](/html-minifier) | [https://html-minifier-v2.now.sh](https://html-minifier-v2.now.sh) | A simple HTML minification example        |
 | [OptiPNG](/optipng) | [https://optipng-v2.now.sh](https://optipng-v2.now.sh) | A PNG optimize example using OptiPNG      |
 | [docz](/docz) | [https://docz.now.sh](https://docz.now.sh) | A documentation example built with [docz](https://docz.site)       |
+| [Gridsome](/docz) | https://gridsome.now.sh/ | Vue-powered static site generated by [Gridsome](https://gridsome.org)       |
```
Comment (TejasQ) [inline README.md]: Careful.

```suggestion
 | [Gridsome](/gridsome) | https://gridsome.now.sh/ | Vue-powered static site generated by [Gridsome](https://gridsome.org)       |
```

File: gridsome/.gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+*.log
```
Comment (TejasQ) [inline gridsome/.gitignore]: Would a root-level gitignore not be more useful in this repo?

File: gridsome/.nowignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (TejasQ) [inline gridsome/.nowignore]: Would a root-level nowignore not be more useful in this repo?

File: gridsome/src/main.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import DefaultLayout from '~/layouts/Default.vue'
```
Comment (TejasQ) [inline gridsome/src/main.js]: The tilde might need explanation for newcomers.

File: gridsome/src/pages/About.vue [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<template>
+  <Layout>
+    <h1>About us</h1>
+    <p>This is the second page of the Gridsome example deployed with Now!</p>
+    <g-link to="/">Back home</g-link>
+  </Layout>
+</template>
+
+<script>
+export default {}
```
Comment (TejasQ) [inline gridsome/src/pages/About.vue]: This can be ambiguous.

Comment (timothyis) [reply gridsome/src/pages/About.vue]: This is generated by Gridsome, I didn't want to touch it too much.

File: gridsome/src/main.js (hunk 4 above)
Comment (timothyis) [reply gridsome/src/main.js]: This is generated by Gridsome, I didn't want to touch it too much.

File: gridsome/.nowignore (hunk 3 above)
Comment (timothyis) [reply gridsome/.nowignore]: This is so that people that clone the directory for themselves have the `.nowignore`

File: gridsome/.gitignore (hunk 2 above)
Comment (timothyis) [reply gridsome/.gitignore]: This is generated by Gridsome, I didn't want to touch it too much.

**Pull request merged**